utilities/blob_db/blob_db_impl.cc

```diff
@@ -2244,6 +2221,39 @@ Iterator* BlobDBImpl::NewIterator(const ReadOptions& opts,
                             column_family, this);
 }
 
+Status DestroyBlobDB(const std::string& dbname, const Options& options,
+                     const BlobDBOptions& bdb_options) {
+  const ImmutableDBOptions soptions(SanitizeOptions(dbname, options));
+  Env* env = soptions.env;
+
+  Status status;
+  std::string blobdir;
+  blobdir = (bdb_options.path_relative) ? dbname + "/" + bdb_options.blob_dir
+                                        : bdb_options.blob_dir;
+
+  std::vector<std::string> filenames;
+  env->GetChildren(blobdir, &filenames);
+
+  for (const auto& f : filenames) {
+    uint64_t number;
+    FileType type;
+    if (ParseFileName(f, &number, &type) && type == kBlobFile) {
+      Status del = env->DeleteFile(blobdir + "/" + f);
+      if (status.ok() && !del.ok()) {
+        status = del;
```

Comment:
I'm trying to match what `DestroyDB()` is doing: https://github.com/facebook/rocksdb/blob/master/db/db_impl.cc#L2229-L2231
I think the idea is to have `DestroyDB()` try to delete every file of the database regardless of errors.